src/components/Homepage/LendByCategory/LoanCategoriesSection.vue

```diff
@@ -138,41 +164,74 @@ export default {
 					query: loanChannelData,
 					variables: {
 						ids: [categoryId],
-						excludeIds: this.allLoanIdsCurrentlyVisible,
+						excludeIds: this.allFetchedLoanIds, // exclude loans that have already been fetched
 						imgDefaultSize: 'w480h300',
 						imgRetinaSize: 'w960h600',
 						numberOfLoans: 8,
 					}
 				}).then(({ data }) => {
-					const channelData = _get(data, 'lend.loanChannelsById')
-						.filter(channel => channel.id === categoryId);
-					const channelLoans = channelData[0];
+					const channelLoans = _get(data, 'lend.loanChannelsById')[0];
 					this.categoriesWithLoans.push(channelLoans);
+
+					// TODO
+					// if we have less than 8 loans left after filtering:
+					// this.getCategoryLoans(categoryId)
+					// fetch more loans
 				});
 			}
 		},
-		// get category loans for a given category id
+		// get filtered loans for a given category id
 		getCategoryLoans(categoryId) {
-			return this.categoriesWithLoans.filter(category => category.id === categoryId)[0] || {};
+			let filteredLoansArray = [];
+
+			const allLoansForCategory = this.categoriesWithLoans
+				.filter(category => category.id === categoryId)[0] || {};
+
+			const testFilter = loan => loan.loanAmount.includes('2');
```

Comment:
What are you testing for here, was this literally a "test" filter? Seems like loanAmount is a string that can include all numbers and a period.

Reply:
I forgot to remove that test filter 🤦